guild.go

```diff
@@ -747,7 +747,7 @@ type GuildQueryBuilder interface {
 	// GetRoles
 	// TODO: For GetRoles, it might sense to have the option for a function to filter before each role ends up deep copied.
 	// TODO-2: This could be much more performant in larger guilds where this is needed.
-	// TODO-3: Add GetRole.
```

Comment:
I see my comment has misguided you, that's on me. Could you adjust this so it's `Client.Guild(id).Role(id).Get()` instead?

Reply:
Essentially you just move the new method to the `GuildRoleQueryBuilder` interface instead, and name it `Get`.